domain-models-runtime/src/main/java/org/folio/rest/persist/PostgresClient.java

```diff
@@ -885,50 +885,55 @@ public void saveBatch(String table, JsonArray entities, Handler<AsyncResult<Resu
   }
 
   public void saveBatch(String table, JsonArray entities, String column, Handler<AsyncResult<ResultSet>> replyHandler) {
+    client.getConnection(res -> {
+      if (res.failed()) {
+        replyHandler.handle(Future.failedFuture(res.cause()));
+        return;
+      }
+      saveBatch(res, table, entities, column, x -> {
+        res.result().close();
+        replyHandler.handle(x);
+      });
+    });
+  }
+
+  public void saveBatch(AsyncResult<SQLConnection> sqlConnection, String table, JsonArray entities, String column, Handler<AsyncResult<ResultSet>> replyHandler) {
```

Comment:
Plz add another wrapper public method, it is more convenient for work with List<Object>.
as:
`public void saveBatch(AsyncResult<SQLConnection> sqlConnection, String table, List<Object> entities, Handler<AsyncResult<ResultSet>> replyHandler) {....}`

In most cases, we operate on entities, and I think it is more convenient to have two methods with List of Objects and an JsonArray